content/300-guides/500-other/880-troubleshooting-orm/100-help-articles/600-vercel-caching-issue.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: 'Vercel deployment caching workaround'
+metaTitle: 'Learn to configure your build process on Vercel to avoid caching-related problems'
+metaDescription: 'Learn to configure your build process on Vercel to avoid caching-related problems'
+---
+
+## Problem
+
+If you deploy an application using Prisma to [Vercel](https://vercel.com/), you may run into strange issues as you iterate on your Prisma schema and deploy updates to your app.
+
+These issues occur because of the fact that Vercel caches the dependencies of your project until one of those dependencies changes. It does this to allow faster builds, and while this is typically a good thing, it causes some problems for Prisma Client.
+
+Prisma uses a `postinstall` hook to generate Prisma Client when dependencies are installed. Because Vercel uses cached modules, this `postinstall` hook never gets run during a deployment after the initial deployment. This results in Prisma Client becoming out of sync with your database schema.
```

Comment:
> ... So we implemented this error message to prevent this situation from happening and this error message directs you here instead to learn how to fix this.